packages/client-core/src/systems/LoadingUISystem.tsx

```diff
@@ -124,8 +164,14 @@ export default async function LoadingUISystem(world: World) {
     // }
 
     const loadingState = getState(LoadingSystemState).loadingScreenOpacity
+    defaultColor.set(getAppTheme()!.textColor)
+    mainThemeColor.set(themeColors.alternate)
+    xrui.rootLayer.traverseLayersPreOrder((layer: WebLayer3D) => {
+      const mat = layer.contentMesh.material as MeshBasicMaterial
+      mat.color.lerpColors(defaultColor, mainThemeColor, engineState.loadingProgress.value * 0.01)
+    })
 
-    transition.update(world.deltaSeconds, (opacity) => {
+    /*transition.update(world.deltaSeconds, (opacity) => {
```

Comment:
This has been temporarily commented out for debugging, to prevent the loading screen from fading out.